extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -586,6 +586,50 @@ an interaction with the calendar view, i.e. when the overlay would open.
 The date picker triggers the  `deactivate` event when the user ends
 their interaction with the calendar view, i.e. when the overlay would close.
 
+##### select
```

Comment:
Can you add a code example of how you would react to a `select` event?

Reply:
Good idea, thanks!